### PR TITLE
Author page

### PR DESCRIPTION
**Pull request by QaysAyad**
Finishing the Author page with mock data

### CODE REVIEW

**Comment by QaysAyad** [comment]
Finishing the Author page with mock data
Screen Shot 2020-08-06 at 6 46 44 PM

File: src/pages/AuthorPage/AuthorPage.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            </MDBCol>
+          </MDBRow>
+        </MDBContainer>
+      </div>
+      <div className="text-center mt-5">
+        <h1 className="text-3xl font-medium inline border-b border-gray-400 px-5">
+          {authorBookText}
+        </h1>
+      </div>
+    </>
```
Comment (MohammedAlabd) [inline src/pages/AuthorPage/AuthorPage.js:65]: just refactor these lines of code into more than one component 
the page file should only call the component and you don't have to write a test for it cuz you will write test for each component

Comment (QaysAyad) [reply src/pages/AuthorPage/AuthorPage.js:65]: you mean the title and paragraph right?

Comment (MohammedAlabd) [reply src/pages/AuthorPage/AuthorPage.js:65]: title and paragraph could be a component and  the social media links could be the other one

Comment (QaysAyad) [reply src/pages/AuthorPage/AuthorPage.js]: Done

**Pull request merged**